content/posts/jellyfin-10-6-0.md

```diff
@@ -0,0 +1,112 @@
+
+Other improvements to the web client include a rewritten image viewer ([jf-web#967](https://github.com/jellyfin/jellyfin-web/pull/967)), a configuration option for the number of items per page in libraries ([jf-web#983](https://github.com/jellyfin/jellyfin-web/pull/983)), a toggle for the nightly version of the Chromecast client ([jf-web#1242](https://github.com/jellyfin/jellyfin-web/pull/1242)), and support for multiple plugin repositories ([jf-web#1393](https://github.com/jellyfin/jellyfin-web/pull/1393), [jf#3244](https://github.com/jellyfin/jellyfin/pull/3244)).
+
+# Patreons and Github Sponsors
+
+With the increased amount of activity on the project, we would like to give a signal boost to some ways to support the people working daily on Jellyfin.
+
+We want to stress that, while some of our contributors individually accept financial donations, Jellyfin and its features will **never** be hidden behind a paywall. Supporting the developers financially is entirely voluntary and won't give you any exclusive access to features or support, nor will it change the priority of your feature requests or issues.
+
+[anthonylavado](https://github.com/anthonylavado), our wonderful PR and developer relation person, accepts donations through [Github Sponsors](https://github.com/sponsors/anthonylavado).
+
+[barronpm](https://github.com/barronpm), whose main work is on the server portion of Jellyfin and who has been spearheading the migration to EF Core has recently opened [a Patreon page](https://www.patreon.com/barronpm).
+
+[dkanada](https://github.com/dkanada/), whose work spread from server to web client, accepts donations through [Github Sponsors](https://github.com/sponsors/dkanada)
+
+[ferferga](https://github.com/ferferga), web client contributor, localization aficionado and all-around awesome person accepts donations through [Github Sponsors](https://github.com/sponsors/ferferga).
+
+[MrTimscampi](https://github.com/MrTimscampi), whose main work has been cleaning up and modernizing the web client, also recently opened [a Patreon page](https://www.patreon.com/mrtimscampi).
+
+[oddstr13](https://github.com/oddstr13), one of the developers of Jellyfin for Kodi is also accepts donations on [Github Sponsors](https://github.com/sponsors/oddstr13).
+
+[thornbill](https://github.com/thornbill) and [nielsvanvelzen](https://github.com/nielsvanvelzen), keepers of the Android clients, can also be supported on Github Sponsors. [Here](https://github.com/sponsors/thornbill) for [thornbill](https://github.com/thornbill) and [here](https://github.com/sponsors/nielsvanvelzen) for [nielsvanvelzen](https://github.com/nielsvanvelzen)
```

Comment:
Seems a bit weird to me that they don't get separate lines when everyone else did 😛

Reply:
Indeed. I was running out of descriptors,  mainly lol

Reply:
Fixed as of d40d70c.